cli_tools/gce_vm_image_import/importer/inflater.go

```diff
@@ -194,23 +195,20 @@ type persistentDisk struct {
 	isUEFICompatible bool
 }
 
-type inflationInfo struct {
+type shadowTestFields struct {
 	// Below fields are for shadow API inflation metrics
 	checksum      string
 	inflationTime time.Duration
 	inflationType string
 }
 
-func createInflater(args ImportArguments, computeClient daisyCompute.Client, storageClient storage.Client, inspector imagefile.Inspector) (inflater, error) {
-	di, err := createDaisyInflater(args, inspector)
-	if err != nil {
-		return nil, err
-	}
-
-	return di, nil
+func createInflater(args ImportArguments, computeClient daisyCompute.Client, storageClient storage.Client,
+	inspector imagefile.Inspector, loggableBuilder *service.SingleImageImportLoggableBuilder) (Inflater, error) {
+	return CreateDaisyInflater(args, inspector)
 }
 
-func createDaisyInflater(args ImportArguments, fileInspector imagefile.Inspector) (inflater, error) {
+// CreateDaisyInflater returns an Inflater implementation that uses a Daisy workflow.
+func CreateDaisyInflater(args ImportArguments, fileInspector imagefile.Inspector) (Inflater, error) {
```

Comment:
For "constructor" type funcs, we've been mostly using New...() naming. I guess Create...() was picked up earlier, but it would be nice to have consistent naming as it could lead to confusion otherwise.

Reply:
Updated to `NewDaisyInflater`